frontend/src/app/main/ui/components/context_menu_a11y.cljs

```diff
@@ -52,8 +52,6 @@
   (sm/lazy-validator schema:option))
 
 (mf/defc context-menu*
-  {::mf/props :obj}
```

Comment:
Components with `*` don't need specific metadata for props, they defaults to `::mf/props :obj`